src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties

```diff
@@ -293,6 +293,7 @@ RSC_019=EPUBs with Multiple Renditions should contain a META-INF/metadata.xml fi
 RSC_020='%1$s' is not a valid URI.
 RSC_021=A Search Key Map Document must point to Content Documents ('%1$s' was not found in the spine).
 RSC_022=Cannot check image details (requires Java version 7 or higher).
+RSC_023='%1$s' is not a valid URL.
```

Comment:
I wouldn't say "is not a valid URL", since technically it is (it's just that the host component is empty).
What about something more explicit along the lines of: `URL '%1$s' has no host component`? (or, maybe more user friendly.